doc/source/cluster/kubernetes/user-guides/logging.md

```diff
@@ -245,16 +245,20 @@ Learn how to write filters in this [filter pattern syntax doc](https://docs.aws.
 (redirect-to-stderr)=
 ## Redirecting Ray logs to stderr
 
-By default, Ray writes logs to files under the ``/tmp/ray/session_*/logs`` directory. If you prefer to redirect logs to stderr of the host pods instead, set the environment variable ``RAY_LOG_TO_STDERR=1`` on all Ray nodes. This practice is not recommended but may be useful if your log processing tool only captures log records written to stderr.
+By default, Ray writes logs to files in the `/tmp/ray/session_*/logs` directory.
+If your log processing tool is capable of capturing log records written to stderr, you can redirect Ray logs to the stderr stream of Ray containers by setting the environment variable `RAY_LOG_TO_STDERR=1` on all Ray nodes.
 
-```{admonition} Alert
+```{admonition} Alert: this practice isn't recommended.
 :class: caution
-There are known issues with this feature. For example, it may break features like {ref}`Worker log redirection to Driver <log-redirection-to-driver>`. If those features are wanted, use the {ref}`Fluent Bit solution <kuberay-fluentbit>` above.
 
-For Clusters on VMs, do not redirect logs to stderr. Follow {ref}`this guide <vm-logging>` to persist logs.
+If `RAY_LOG_TO_STDERR=1` is set, Ray will not write logs to files.
```

Comment:
```suggestion
If `RAY_LOG_TO_STDERR=1` is set, Ray doesn't write logs to files.
```

Reply:
I just found out that Vale does not work for the admonition block.